Plugins/BTCPayServer.RockstarDev.Plugins.TransactionCounter/Views/PublicCounter/Counter.cshtml

```diff
@@ -0,0 +1,131 @@
+
+        const counter = document.getElementById('tx-count');
+
+        const updateCounter = () => {
+            const progress = frame / totalFrames;
+            const currentCount = Math.round(finalCount * easeOutQuad(progress));
+            counter.textContent = currentCount.toLocaleString();
+            if (frame < totalFrames) {
+                frame++;
+                requestAnimationFrame(updateCounter);
+            }
+        };
+        updateCounter();
+    </script>
+}
+else
+{
+    <!DOCTYPE html>
+    <html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Store Transactions</title>
+        <style>
+            html, body {
+                margin: 0;
+                padding: 0;
+                overflow: hidden;
+                height: 100%;
+                font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+            }
+
+            video.bg-video {
+                position: fixed;
+                right: 0;
+                bottom: 0;
+                min-width: 100%;
+                min-height: 100%;
+                z-index: -1;
+                object-fit: cover;
+            }
+
+            .counter-box {
+                position: absolute;
+                top: 50%;
+                left: 50%;
+                transform: translate(-50%, -50%);
+                text-align: center;
+                font-size: 4em;
+                font-weight: bold;
+                color: white;
+                text-shadow: 0 0 10px #000;
+                animation: fadeIn 2s ease-out;
+            }
+
+            @@keyframes fadeIn {
+                from {
+                    opacity: 0;
+                    transform: translate(-50%, -40%);
+                }
+
+                to {
+                    opacity: 1;
+                    transform: translate(-50%, -50%);
+                }
+            }
+        </style>
+    </head>
+    <body>
+    <video class="bg-video" autoplay muted loop playsinline>
+        <source src="@videoUrl" type="video/mp4">
+        Your browser does not support the video tag.
+    </video>
+
+    <div class="counter-box">
+        <span id="tx-count">0</span>
+    </div>
+
+    <script>
+        const finalCount = @finalCount;
+        const duration = finalCount <= 10 ? 1000 : 2000000;
+        const frameRate = 60;
+        const totalFrames = Math.round(duration / (1000 / frameRate));
+        let frame = 0;
+        const easeOutQuad = t => t * (2 - t);
+
+        const counter = document.getElementById('tx-count');
+
+        const updateCounter = () => {
+            const progress = frame / totalFrames;
+            const currentCount = Math.round(finalCount * easeOutQuad(progress));
+            counter.textContent = currentCount.toLocaleString();
+            if (frame < totalFrames) {
+                frame++;
+                requestAnimationFrame(updateCounter);
+            }
+        };
+        updateCounter();
```

Comment:
Please update the JavaScript code to fetch the number of transactions from the server and display the counter. The example page I provided was simulating a counter increment. We need to remove that functionality and instead show the number of paid invoices from all stores that we retrieve from the API.